controller/lib/pid/pid.cpp

```diff
@@ -98,7 +111,10 @@ bool PID::Compute() {
 
     /*Remember some variables for next time*/
     lastInput = input;
-    lastTime = now;
+    lastError = error;
+    lastTime += SampleTime; // this makes the PID able to somewhat "catch up"
```

Comment:
Note to self, revisit this when I better understand what each of these two variables means... And question, will this be able to catch up if it loses multiple samples?

Reply:
> will this be able to catch up if it loses multiple samples?
It should (will run on every call until lastTime catches up to now) but I think so long as we keep calling it faster than sample time, we shouldn't worry too much about missing a sample, let alone multiple ones.